scripts/install

```diff
@@ -28,8 +28,7 @@ function check_strimzi {
 
 function deps {
     info "Installing Helm dependencies"
-    helm repo add google https://kubernetes-charts.storage.googleapis.com
```

Comment:
Anyone installing hindsight for the first time will need this line for the script to work, won't they?

Reply:
changing the `helm dependency build` to `helm dependency update` takes care of this. if we were installing anything from the stable repo via `helm upgrade --install foo bar.com` the way we do with Strimzi we would, but since the only things we need from the stable repo are being installed as _part of hindsight_ dependencies this isn't required.

Reply:
to put it a different way, if we're _only_ including charts from that repo as explicit dependencies of hindsight's chart, this separate line isn't necessary

Reply:
👍

Reply:
Thanks for the clarification